### ISSUE TITLE
Incorrect classification for Israel

### ISSUE THREAD

**Issue by yotamN** [osm-without-borders/cosmogony#85]
Israel have a complicated way to define the administration borders so I want to ask how to handle it before opening a PR.

Israel as a country have the following hierarchy:
 - Country (Israel)
 - Districts (Northern District, Southern District, etc), they are shown as states in cosmogony.
 - Council (City council, local council and regional council), they are mapped as cities.
 - Villages which are only under a regional council

So basically `admin_level=8` can mean either a city council/local council which are actually a city but they can also mean a regional council which is a collection of villages or kibbutzim.

Sometimes the council will be marked as `place=city/town` and sometimes inside the council area there would be another area that is tagged as `place=city/town/village`.

Maybe using the `place` tag would be better?
